### PR TITLE
webui: fix services display

### PR DESCRIPTION
**Pull request by crcro**
DO NOT DELETE THIS TEXT

#### Please note

> Please read this information carefully. You can run `./scripts/pre-commit.php` to check your code before submitting.

- [x] Have you followed our [code guidelines?](http://docs.librenms.org/Developing/Code-Guidelines/)

#### Testers

If you would like to test this pull request then please run: `./scripts/github-apply <pr_id>`, i.e `./scripts/github-apply 5926`

fixed display bug when only one host had services

before:
screen shot 2018-01-01 at 4 35 35 am

after:
screen shot 2018-01-01 at 4 35 01 am

### CODE REVIEW

File: html/pages/services.inc.php [hunk 1]
```diff
@@ -35,12 +35,7 @@
                     $vars['view'] = 'basic';
                 }
 
-                $status_options = array(
```
Comment (laf) [inline html/pages/services.inc.php]: I think this is an unnecessary change. It's a bit more readable as it was .

File: html/pages/services.inc.php [hunk 2]
```diff
@@ -141,9 +138,14 @@
                     } else {
                         $sql_param[0] = $device_id;
                     }
-                    $head = true;
 
-                    foreach (dbFetchRows("SELECT * FROM `services` WHERE `device_id` = ? $where ORDER BY service_type", $sql_param) as $service) {
+                    $header = true;
+                    $footer = false;
+
+                    $svc = 0;
```
Comment (laf) [inline html/pages/services.inc.php]: I'd like to see better variable names that describe what data they contain. Same for the two below.

**Comment by laf** [comment]
I must be missing something here, what does this actually change as visually I can't see a difference between the old and the new with this on my test install? I can see the difference on your screenshots though. 

The current demo.librenms.org also doesn't show how it looks in your old screenshot.

**Comment by crcro** [comment]
it only showed like that when services where critical and for one host only ...

**Comment by laf** [comment]
I still don't see the difference with 1 host or 4.

This also breaks the dynamic removal of the service, it was fixed again recently so you'll need to rebase and make sure that these fixes aren't removed: https://github.com/librenms/librenms/pull/8030

**Comment by crcro** [comment]
already rebased and included #8030 fix

**Comment by laf** [comment]
@crcro Can you show a broken screenshot as I'm not convinced this is changing anything visually.

**Comment by crcro** [comment]
@laf this is from a up2date librenms install

screen shot 2018-01-25 at 11 02 18 pm

**Comment by murrant** [comment]
@crcro How it looks in master for me:

image

**Comment by crcro** [comment]
@murrant how does it look if you click on "critical" only? on all, it shows good for me also
